packages/conf-llvm/conf-llvm.3.9/files/source-linux.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+sudo add-apt-repository 'deb http://llvm.org/apt/trusty/ llvm-toolchain-trusty-3.9 main'
+curl -L http://llvm.org/apt/llvm-snapshot.gpg.key | sudo apt-key add -
```

Comment:
Note that you don't necessarily have to make everything green as long as an explanation can be provided (it may be good to add a comment in the opam file to indicate why this is the case). Maybe it's something the depext redesign should allow to take into account.